extensions/plugins/system/redcore/redcore.php

```diff
@@ -344,6 +344,23 @@ public function onBeforeRender()
 	}
 
 	/**
+	 * Called before a JForm is rendered. It can be used to modify the JForm object in memory before rendering.
+	 * https://docs.joomla.org/Plugin/Events/Content
+	 *
+	 * @return  void
+	 */
+	public function onContentPrepareForm()
+	{
+		// If the options to do so are turned on, create a button for opening a modal window to edit translations directly from a translatable form
+		if ($this->params->get('enable_translations') == 1 && $this->params->get('show_edit_button_on_all_forms') == 1)
```

Comment:
you should go here for default values like $this->params->get('show_edit_button_on_all_forms', 0) == 1